### PR TITLE
fixed issue with v2 api not on user search

### PR DESCRIPTION
**Pull request by giordyb**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] Ready

## Related Issues
fixes: [link to the issue](https://panw-global.slack.com/archives/C011E5T5K17/p1640600470020600)

## Description
Just a couple of changes to fix the issue on the *search_user* and *get_account_id_from_attribute* functions where the REST APIs used on the On-Prem version of Jira (Jira Server) differs from the one used in the cloud (v2 vs v3)
## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - [X ] No

### CODE REVIEW

**Comment by bakatzir** [comment]
@yuvalbenshalom FYI for the on-prem/cloud API disparity, please consider reviewing.

**Comment by dorschw** [comment]
Hi @giordyb, thanks for your contribution,
Please [sign the SLA](https://github.com/demisto/content/pull/16667#issuecomment-1001589510), and [add a unit test](https://xsoar.pan.dev/docs/integrations/code-conventions#unit-tests) for the change you applied in `jira_v2_test.py`.

File: Packs/Jira/ReleaseNotes/1_4_5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- %%UPDATE_RN%%
```
Comment (dorschw) [inline Packs/Jira/ReleaseNotes/1_4_5.md:4]: Please describe the change here, you can find more information [here](https://xsoar.pan.dev/docs/documentation/release-notes)

**Comment by giordyb** [comment]
> Hi @giordyb, thanks for your contribution, Please [sign the SLA](https://github.com/demisto/content/pull/16667#issuecomment-1001589510), and [add a unit test](https://xsoar.pan.dev/docs/integrations/code-conventions#unit-tests) for the change you applied in `jira_v2_test.py`.

@dorschw I don't see any existing unit tests for the 2 functions I made changes to.

**Comment by dorschw** [comment]
> > Hi @giordyb, thanks for your contribution, Please [sign the SLA](https://github.com/demisto/content/pull/16667#issuecomment-1001589510), and [add a unit test](https://xsoar.pan.dev/docs/integrations/code-conventions#unit-tests) for the change you applied in `jira_v2_test.py`.
> 
> @dorschw I don't see any existing unit tests for the 2 functions I made changes to.

You're right, thanks.

**Comment by giordyb** [comment]
@dorschw FYI I made a mistake where my original commits got pushed with a different email address, that's why you see 2 committers. 
I thought I corrected the issue with a re-commit but it still shows the wrong committer. I already accepted the licensing agreement with the correct email address please let me know if this works for you or if you want me to re-submit.

**Comment by dorschw** [comment]
> @dorschw FYI I made a mistake where my original commits got pushed with a different email address, that's why you see 2 committers. I thought I corrected the issue with a re-commit but it still shows the wrong committer. I already accepted the licensing agreement with the correct email address please let me know if this works for you or if you want me to re-submit.

thanks for the update, don't worry about it, we'll take care of it.

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
@giordyb following our slack conversation, please remove the parameter `jira_v2_api` and instead add an argument "JIRAv2API" to `ira-get-id-by-attribute` and use it instead.
make this argument false by default.

**Review by yuvalbenshalom — APPROVED** [review]
approving the fix
@dorschw / @DeanArbel please see description and release notes if they need a some editing.

**Comment by DeanArbel** [comment]
@yaron-libman Please assign someone from the technical writers team to review the new argument: `is_jirav2api`.
We should also add these changes to the pack `Readme.md` (if you have issues doing so, please update me, and I'll assist with it).

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 2]
```diff
@@ -7,109 +7,130 @@ configuration:
...
   required: false
   type: 8
-- display: Incident type
-  name: incidentType
-  required: false
-  type: 13
-- display: Use created field to fetch incidents
+  defaultvalue: "false"
+  additionalinfo:
+- defaultvalue: "true"
```
Comment (DeanArbel) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:83]: Please revert - you can change to `false`.
```suggestion
- defaultvalue: "false"
```

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 3]
```diff
@@ -7,109 +7,130 @@ configuration:
...
-  hidden: false
   name: outgoing_mirror
   required: false
   type: 8
-- additionalinfo: Choose the tag to add to an entry to mirror it as an attachment
-    in Jira.
-  defaultvalue: attachment
-  display: File entry tag
-  hidden: false
+  defaultvalue: "true"
```
Comment (DeanArbel) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:93]: Please revert - you can change it to `false`.

```suggestion
  defaultvalue: "false"
```

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 4]
```diff
@@ -7,109 +7,130 @@ configuration:
...
+  additionalinfo: Choose the tag to add to an entry to mirror it as an attachment
+    in Jira.
+- display: Comment entry tag
   name: comment_tag
   required: false
   type: 0
+  defaultvalue: comment
+  additionalinfo: Choose the tag to add to an entry to mirror it as a comment in Jira.
 - additionalinfo: Fetch comments for Jira ticket.
+  defaultvalue: "true"
```
Comment (DeanArbel) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:109]: Please revert - you can change it to `false`.
```suggestion
  defaultvalue: "false"
```

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 5]
```diff
@@ -7,109 +7,130 @@ configuration:
...
   display: Fetch comments
-  hidden: false
   name: fetch_comments
   required: false
   type: 8
 - additionalinfo: Fetch attachments for Jira ticket.
+  defaultvalue: "true"
```
Comment (DeanArbel) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:115]: Please revert - you can change it to `false`.
```suggestion
  defaultvalue: "false"
```

**Review by DeanArbel — CHANGES_REQUESTED** [review]
Thank you very much for this contribution. The PR is almost ready, and is mostly pending our technical writers team review.

Please note that you'll need to revert some changes you've committed to the integration parameters. I have collected the changes with a suggestion below.

**Comment by giordyb** [comment]
@DeanArbel I went ahead and reverted the changes to JiraV2.yaml and added only the changes necessary.

**Comment by julieschwartz18** [comment]
@DeanArbel - Doc review complete.

**Review by DeanArbel — APPROVED** [review]

**Comment by DeanArbel** [comment]
@giordyb I have approved the PR, and will assist you in updating the readme.
The update to the pack should be available by tomorrow.

Thank you very much for your contribution!

**Pull request merged**